console/src/components/Services/ApiExplorer/ApiExplorer.scss

```diff
@@ -827,3 +827,8 @@ label {
     }
   }
 }
+
+.graphiqlModeToggle {
+  margin-top: -5px;
```

Comment:
Can we instead decrease margin-top here: https://github.com/hasura/graphql-engine/blob/master/console/src/components/Common/Common.scss#L1264?
So that those two elements don't overlap.

Reply:
I don't understand. Can you elaborate?

Reply:
Sure. A few points on negative margins:
* Layout is not clean — elements overlap 
<img width="193" alt="Screenshot 2020-06-04 at 08 50 21" src="https://user-images.githubusercontent.com/9019397/83724353-6e7c9e80-a640-11ea-9b20-419f10a36721.png">

* When composing a layout we should stick to make it as simple as possible 

* Negative margins may lead to inconsistent results in different browsers (might not be the case for us, tho)

Sometimes negative margins are needed and there's no other way around, but in this case, we can compose the elements by reducing the margin of one of them. That's why I put this comment. 
This is not something super bad but the thing is they may be harmful, so it's good to avoid them when possible.